deployments/liqo/templates/webhooks/liqo-validating-webhook.yaml

```diff
@@ -8,6 +8,23 @@ metadata:
   labels:
     {{- include "liqo.labels" $webhookConfig | nindent 4 }}
 webhooks:
+  - name: fc.validate.liqo.io
+    admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      service:
+        name: {{ include "liqo.prefixedName" $ctrlManagerConfig }}
+        namespace: {{ .Release.Namespace }}
+        path: "/validate/foreign-cluster"
+        port: {{ .Values.webhook.port }}
+    rules:
+      - operations: ["CREATE", "UPDATE"]
```

Comment:
If I understand correctly we are only handling update events

Reply:
Definitely. I currently kept both operations since it is likely we will add checks that apply also to creations in the future. Yet, I'm open to remove the "CREATE" one for the moment if you feel it is better.

Reply:
Yes, I think that we can remove it for now